types/javascript-color-gradient/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for javascript-color-gradient 1.3
+// Project: https://github.com/Adrinlol/javascript-color-gradient
+// Definitions by: Wilson Chua <https://github.com/wilsoncwc/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = Gradient;
+
+declare class Gradient {
+    constructor(gradients?: string, maxNum?: number, color?: string[], intervals?: any[]);
+    setGradient(...args: string[]): Gradient;
```

Comment:
should this be:
```ts
setGradient(color1: string, color2: string, ...otherColors: string[]):  this;
```

Reply:
Yep that works nicely, thanks!

```suggestion
    setGradient(color1: string, color2: string, ...otherColors: string[]):  this;
```